etc/current_env_content.txt

```diff
@@ -0,0 +1,2 @@
+kw_show_current_env() { kw_env_file=$(git rev-parse --show-toplevel 2>/dev/null)/.kw/env.current; [ -f "$kw_env_file" ] && printf '[%s]' "$(cat "$kw_env_file")"; }
```

Comment:
Why this code need to looks like this? Can you put it as a standard function in a readable way? Also, use the `.sh` extension or no extension. Finally, this file name does not describe this script well... how git names something like this?

Reply:
I had compacted this code into two lines so as not to take up too much space in the code, but now moving it to etc makes it easier to extend the code again. As for the name, I'll think of something better.

Reply:
Using the readable/extended code, it isn't very easy to check if this code already exists inside the _.bashrc_. Could you give me a recommendation?

![image](https://user-images.githubusercontent.com/43859386/226380491-357cefe9-2779-4fed-b0c4-5894939ff637.png)

Reply:
I don't think you should inject `kw_get_current_env_name()` in the bashrc this way. Keep this code in a file under kw folder, and just include it in the bashrc.

Reply:
Also, you should not use `PS1=`; this will override any previous PS1 configuration. It would be best if you first ensure that the `kw_get_current_env_name` is available; after that, you need should use PS1+=.